<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Split-It</string>
  </property>
  <property name="windowIcon">
   <iconset resource="r.qrc">
    <normaloff>:/123.svg</normaloff>:/123.svg</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditDestination"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="destinationButton">
            <property name="text">
             <string>Destination</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="sourceButton">
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditSource"/>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="splitButton">
            <property name="text">
             <string>Split</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Max rows quantity in file</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QLineEdit" name="lineEditRowsQuantity">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>56</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>250000</string>
              </property>
              <property name="maxLength">
               <number>6</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QCheckBox" name="checkBoxAddFirstRow">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>164</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Add first row to result files</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QCheckBox" name="checkBoxStrictSplit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Strict Split</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>94</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetMain"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Cell delimetr</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxCell">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>,</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>;</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Text delimetr</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxText">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>&quot;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>'</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxMultipleRows">
            <property name="text">
             <string>Duplicate rows with more than one email in row</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxRemoveRowsWithoutEmails">
            <property name="text">
             <string>Remove rows without emails</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonShowPreview">
            <property name="text">
             <string>Show Preview</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButtonReloadSourceFile">
            <property name="text">
             <string>Reload Source File</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Manual</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetPreview"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
   </widget>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionhueraga_1_0_Sergei_Vinogradov">
   <property name="text">
    <string>hueraga 1.0, Sergei Vinogradov</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEditSource</tabstop>
  <tabstop>lineEditDestination</tabstop>
  <tabstop>sourceButton</tabstop>
  <tabstop>destinationButton</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>tableWidgetMain</tabstop>
  <tabstop>checkBoxMultipleRows</tabstop>
  <tabstop>checkBoxRemoveRowsWithoutEmails</tabstop>
  <tabstop>comboBoxCell</tabstop>
  <tabstop>comboBoxText</tabstop>
  <tabstop>pushButtonReloadSourceFile</tabstop>
  <tabstop>tableWidgetPreview</tabstop>
 </tabstops>
 <resources>
  <include location="r.qrc"/>
 </resources>
 <connections/>
</ui>
